src/system/router.py

```
@@ -0,0 +1,39 @@
+def info():
+    """
+    Returns information about the service.
+
+    This endpoint provides details about the service, including its name and version.
+    This endpoint provides details about the service, including its name, version, and environment variables.
+
+    Returns:
+        A JSON object with service information.
+    """
+    return {
+        "host": os.getenv("ENVY_HOST", "0.0.0.0"),
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)